src/main/java/nextstep/ladder/strategy/RandomStrategy.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.ladder.strategy;
+
+import java.util.Random;
+
+public class RandomStrategy implements LiningStrategy {
+    private final Random random = new Random();
+
+    @Override
+    public boolean canDrawLine() {
```

Comment:
can이 동사가 아닌 조동사라.. 무언가 행위를 하는 것으로 느껴지진 않아 상관은 없어 보이는데요 `boolean` 반환 시 일반적으로 사용하는 `is` Prefix를 사용하시는건 어떨까요?

```suggestion
    public boolean isDrawLine() {
```

Reply:
저도 많이 고민한 부분인데요..ㅠㅠ
Draw가 동사라 is가 나오는게 이상해서 can을 사용했습니다.
이 부분은 조금 더 고민해보고 반영하도록 하겠습니다!